src/Text/Layout/Table.hs

```diff
@@ -277,7 +278,7 @@ tableLinesB specs TableStyle { .. } header rowGroups =
 
     -- Optional values for the header
     (addHeaderLines, fitHeaderIntoCMIs, realTopH, realTopL, realTopC, realTopR)
-                  = case header of
+                  = case colHeader of
```

Comment:
It lets you know that the headers are for the columns rather than the rows. I can revert if you'd prefer, but given that row headers are likely to be introduced soon, I think it helps clarify.